src/routes/things.js

```diff
@@ -47,8 +54,62 @@ let router = express.Router();
  * }
  **/
 
-router.get('/Things', (req, res) => {
-  res.status(200).send();
+const resource = 'Things';
+
+router.get('/', (req, res) => {
+  db().then((models) => {
+    models.Things.findAll().then(things => {
+      res.status(200).send(response.generate(things));
```

Comment:
According to [10.2.2](http://docs.opengeospatial.org/is/15-078r6/15-078r6.html#64) the header of the response needs to contain the URL of the created entity, like:

`Location: http://pg-api.sensorup.com/OGCSensorThings/v1.0/Things(276065)`